### PR TITLE
Release v3.13.0

### PR DESCRIPTION
**Pull request by elewis2**
release notes v3.12.3:

Improvements:

- DEVSU-2216: add 'recruitment_status' to kb_data in KbMatch results where statement subject is a clinical trial

### CODE REVIEW

**Review by mathieulemieux — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -18,7 +18,7 @@ known_standard_library = requests
 
 [metadata]
 name = ipr
-version = 3.12.2
```
Comment (dustinbleile) [inline setup.cfg:21]: This is an entirely new feature, right?

I think then, the version would usually be increased to v3.13.0, but that is minor.

**Review by dustinbleile — APPROVED** [review]

**Review by calchoo — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (elewis2) [reply setup.cfg:21]: It is but it is so trivial. You're probably right though, I'll update that.

**Review by dustinbleile — APPROVED** [review]

**Comment by mathieulemieux** [comment]
Nothing to complain about but just pointing out that changes happening after review don't seem to dismiss approval.

**Comment by creisle** [comment]
> Nothing to complain about but just pointing out that changes happening after review don't seem to dismiss approval.

fixed! not sure how that setting got turned off (maybe github updated something? @elewis2 you may want to double check the other repos to make sure its not space-wide problem)

**Pull request merged**